README.md

```diff
@@ -79,9 +114,12 @@ python3 tests/fast_implementation_test.py
 
 ## Tutorial
 
+#### In Google colab
 You can have a look at our [Google Colab notebook](https://colab.research.google.com/github/GeomScale/dingo/blob/develop/tutorials/dingo_tutorial.ipynb)
 on how to use `dingo`.
 
+#### In local environment
+You can have a look at `dingo/tutorials/dingo_walkthrough.ipynb`. Now you should be able to run all cells.
```

Comment:
`dingo/tutorials/dingo_walkthrough.ipynb` is intended for tutorial purpose in local environment after complete installation.  

Not sure what you meant by link to the tutorial file, should I create a colab notebook?

Reply:
I propose to minimize the review process and leave the tutorial as a separate PR. So this PR will only include the changes to README. Do you agree?

Reply:
Also I am not convinced that we need a "local" version of the tutorial. Isn't the current tutorial enough to execute locally?

Reply:
The current tutorial should be sufficient for local execution. My initial thought was to have a separate file specifically for the local project demo, but on second thought, the existing tutorial works perfectly.  I've gone ahead and removed the "dingo_walkthrough" file to avoid any confusion.

Reply:
Sure, Let's keep this PR focused on the README changes and submit the tutorial in a separate PR. This will help keep the review process streamlined.